.github/workflows/build-docs.yml

```diff
@@ -0,0 +1,31 @@
+name: build-docs
+on:
+  pull_request:
+    paths:
+      - .github/workflows/**
+      - docs/**
+  push:
```

Comment:
```suggestion
```

Reply:
I've copied this straight from ci.yml. Are you sure that's redundant?

Reply:
checked other yamls and this really seems redundant